api/src/swagger/config.yaml

```diff
@@ -438,6 +438,8 @@ paths:
       responses:
         '200':
           description: 'Successful query'
+        '404':
+          description: 'Not found'
         '500':
```

Comment:
Shouldn't we remove all 500 in swagger now? I don't think it's possible to trigger that code any longer?

Reply:
I guess it could be triggered if there is an unexpected error during the code execution, for example if the db fails?
See https://github.com/MetabolicAtlas/MetabolicAtlas/blob/main/api/src/endpoints/gotEnzymes.js#L41 . Maybe the explicit returns of `500`s should be removed, though? I can't find similar cases for the other endpoints.

Reply:
Yeah I think so, since it's unlikely that 500 will be triggered from the swagger api examples

Reply:
Removed in 6a297d9